Game2048/Game/Pages/Game2048Page.razor

```diff
@@ -0,0 +1,86 @@
+@page "/"
+
+<div class="container">
+    <div class="heading">
+        <h1 class="title">2048</h1>
+        <div class="scores-container">
+            <div class="score-container">0</div>
+            <div class="best-container">12456</div>
+        </div>
+    </div>
+    <div class="above-game">
+        <p class="game-intro">Join the tiles, get to <strong>2048!</strong><br><a href="javascript:void(0)"
+                class="how-to-play-link">How to play →</a></p><a class="restart-button">New Game</a>
+    </div>
+    <div class="game-container">
+        <div class="game-message">
+            <p></p>
+            <div class="lower"><a class="keep-playing-button">Keep going</a>
+                <a class="retry-button">Try again</a>
+            </div>
+        </div>
+        <div class="grid-container">
```

Comment:
I think you should delete all of this container because the game is going to be encapsulated in another module made by me.